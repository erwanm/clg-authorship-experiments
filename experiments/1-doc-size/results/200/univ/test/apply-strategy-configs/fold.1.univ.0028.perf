0.459624	0.696400	0.660000
