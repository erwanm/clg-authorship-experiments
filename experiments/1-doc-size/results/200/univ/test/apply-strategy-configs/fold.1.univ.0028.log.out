apply-config.sh: applying config '/tmp/200/process//selected-meta-configs/fold.1.strategy-as-meta.univ.0028.model/strategy-configs/fold.1.univ.0028' to cases '/home/moreaue/dump/expe1//results/200/univ/test/apply-strategy-configs/only-cases.tmp.RKZPjIDl'; results written to '/home/moreaue/dump/expe1//results/200/univ/test/apply-strategy-configs/fold.1.univ.0028'
apply-config.sh: done.
